docs/benchmarks/2023-09-13-byconity-tpcds-benchmark-olap/index.md

```diff
@@ -0,0 +1,120 @@
+Thread(s) per core:   |  2
+Core(s) per socket:   |  12
+Socket(s):            |  2
+NUMA node(s):        |   2
+Vendor ID:            |  GenuineIntel
+CPU family:           |  6
+Model:                 | 79
+Model name:            | Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz
+Stepping:           |    1
+CPU MHz:              |  2494.435
+CPU max MHz:          |  2900.0000
+CPU min MHz:         |   1200.0000
+BogoMIPS:            |   4389.83
+Virtualization:      |   VT-x
+L1d cache:           |   32K
+L1i cache:           |   32K
+L2 cache:            |   256K
+L3 cache:             |  30720K
+NUMA node0 CPU(s):   |   0-11,24-35
+NUMA node1 CPU(s):    |  12-23,36-47
+
+Testing method:
+- Tested the performance of 4 OLAP engines with 99 queries and 1 TB (2.80 billion rows) of data using the TPC-DS benchmark.
+- Use the same test dataset in each engine and maintain the same configuration and hardware environment.
+- For each query, execute multiple times and average to reduce measurement errors, setting a timeout of 500 seconds per query.
+- Record the details of query execution, such as query execution plan, I/O, and CPU usage.
+
+## Performance test results
+We conducted performance testing on these four OLAP engines using the same dataset and hardware environment. The dataset size was set at 1TB, and the detailed hardware and software specifications have been provided earlier in this report. To facilitate transparency and accessibility, we've shared the ByConity benchmark framework(https://github.com/ByConity/byconity-tpcds/blob/main/Practice.md) on GitHub.
+
+Our testing involved running 99 query statements from the TPC-DS benchmark on each of the four OLAP engines. We performed three consecutive test runs and calculated the average results for each engine. Notably, ByConity successfully completed all 99 query tests. However, Doris encountered issues, crashing during SQL15 and experiencing timeouts in SQL54, SQL67, SQL78, and SQL95. Presto, on the other hand, only timed out in SQL67 and SQL72, while successfully completing all other query tests.
+
+ClickHouse presented unique challenges as it managed to execute only 50% of the query statements. This performance discrepancy is attributed to potential limitations in effectively handling multi-table associated queries. It may require manual rewriting and query splitting for such SQL statements to execute. Consequently, for the purpose of comparing total execution times, we temporarily excluded ClickHouse from consideration. In Figure 1 below, we provide a visual representation of the total execution times for the remaining three OLAP engines.
+
+Figure 1 illustrates that ByConity's open-source query performance superseds the other engines, achieving approximately 3-4 times faster execution times. (Please note that all units on the vertical axis of the subsequent charts are expressed in seconds).
+
+![Figure 1: TPC-DS 99 query total time](./TPC-DS-query-performance-times.png)
```

Comment:
@vinijaiswal  the image link is not right, I don't think the jpeg picture is the right one. 
Could you replace the image with a new commit?

Reply:
Fixed it by updating the image's URL.